### PR TITLE
Add download buttons from webflow

### PR DESCRIPTION
**Pull request by TomaszKolek**

### CODE REVIEW

File: budgetportal/webflow/templates/webflow/detail_provincial-infrastructure-projects.html [hunk 1]
```diff
@@ -141,7 +141,7 @@
           <div data-w-id="505b95b7-062c-dafb-5a18-96ee222ccfff" class="nav-menu__dropdown_trigger"><img src="/static/images/menu-24px.svg" alt="" class="nav-menu__icon-open"><img src="/static/images/close-24px.svg" alt="" class="nav-menu__icon-close"></div>
           <div class="nav-menu__items">
             <div class="nav-menu__links">
-              <div class="nav-menu__stages"></div><a href="/infrastructure-projects/provincial/" class="nav-link w-nav-link">Project search</a><a href="https://vulekamali.gov.za/" target="_blank" class="nav-link--feature w-nav-link">Visit vulekamali</a></div>
```
Comment (jbothma) [inline budgetportal/webflow/templates/webflow/detail_provincial-infrastructure-projects.html]: Reminder to myself to fix this in webflow

File: budgetportal/webflow/templates/webflow/infrastructure-search-template.html [hunk 2]
```diff
@@ -224,7 +224,7 @@
           <div data-w-id="505b95b7-062c-dafb-5a18-96ee222ccfff" class="nav-menu__dropdown_trigger"><img src="/static/images/menu-24px.svg" alt="" class="nav-menu__icon-open"><img src="/static/images/close-24px.svg" alt="" class="nav-menu__icon-close"></div>
           <div class="nav-menu__items">
             <div class="nav-menu__links">
-              <div class="nav-menu__stages"></div><a href="/infrastructure-projects/provincial/" class="nav-link w-nav-link">Project search</a><a href="https://vulekamali.gov.za/" target="_blank" class="nav-link--feature w-nav-link">Visit vulekamali</a></div>
```
Comment (jbothma) [inline budgetportal/webflow/templates/webflow/infrastructure-search-template.html]: Reminder to fix this in webflow

**Comment by TomaszKolek** [comment]
@jbothma are you able to check what is the problem with WIP code?
The code there causes `Uncaught SyntaxError: Unexpected end of input` error.
What's weird, even if comment that code out it still shows an error.

I tried with getElementByID - the same problem.
Can you please check on your machine?

**Comment by TomaszKolek** [comment]
Maybe I did something wrong in terms of your rule here:
```
It is critical that none of the static files or templates in the webflow app are modified directly. That code is modified in webflow.com and imported here. Direct changes will be overwritten.

The javascript to implement this feature is in static/js/webflow.```

**Comment by TomaszKolek** [comment]
@jbothma what is the solution... change yarn version from 1.21 to 1.22 - thats what I hate FE for! :D

**Pull request merged**